index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=inter">
+    
+      
+</head>
+<body>
+    <header class="navbar">
+        <a class="welcome">Welcome</a>
+        <img class="menu" alt="button" src="Enabled.png" >
+    </header>
+    <div class="main">
+        <h1>Hey there. I’m Amalia</h1>
+        <h2>I’m a software developer</h2>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+    </div>
+    <div class="icons">
+        <img src="twitter.png" alt="twitter"> 
+        <img src="Linkedin.png" alt="Linkedin">
+        <img src="medium.png" alt="medium">
+        <img src="github.png" alt="github">
+        <img src="angellist.png" alt="angellist">
+    </div>
```

Comment:
- [ ] Please make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)